packages/v4/patternfly-docs/content/training/html-css/css-variables-and-overrides.md

```diff
@@ -0,0 +1,227 @@
+---
+id: HTML CSS variables and overrides training
+section: training
+hideNavItem: true
+---
+import { Button, ClipboardCopy } from '@patternfly/react-core';
+
+# CSS variables and overrides
+PatternFly Core is based on the principles of <a href="http://bradfrost.com/blog/post/atomic-web-design/" target="_blank">Atomic design</a> and BEM (Block, Element, Modifier). <a href="http://getbem.com/introduction/" target="_blank">BEM</a> is a popular CSS methodology for building modular, scalable applications. It provides scope, avoids inheritance, and reduces CSS specificity and conflicts.
+
+BEM works perfectly with a modular design system, as each unique component can be represented as an independent block. Since a block is tied to a component, developers are able to develop, move around, and nest components without conflicts in their application’s CSS.
+PatternFly uses a modified version of BEM for its CSS architecture. Where PatternFly deviates from BEM is in relation to modifiers.
+
+This tutorial covers how PatternFly uses BEM as a framework for its component library. You'll learn how to override and create both global and component-level custom properties and component elements.
+
+## Part 1: Overriding PatternFly component-level properties
+In PatternFly, component-level custom properties follow this general formula:
+`--pf-c-block[__element][--modifier][--state][--breakpoint][--pseudo-element]--PropertyCamelCase.`
+ - `--pf-c-block` refers to the block, usually the component or layout name (e.g. `--pf-c-alert`).
```

Comment:
```suggestion
 - `--pf-c-block` refers to the block, usually the component or layout name (for example,`--pf-c-alert`).
```

PF suggests avoiding "e.g." ([here](https://www.patternfly.org/v4/ux-writing/terminology#abbreviations-and-acronyms)), so just tweaking